### PR TITLE
Add hasDataBreakpoint attribute for VariablePresentationHint

### PR DESCRIPTION
**Pull request by isadorasophia**
This adds an 'hasDataBreakpoint' attribute and deprecate 'dataBreakpoint' kind at VariablePresentationHint, per #173.

### CODE REVIEW

File: debugAdapterProtocol.json [hunk 1]
```diff
@@ -3361,23 +3361,24 @@
 						"Indicates that the object is an interface.",
 						"Indicates that the object is the most derived class.",
 						"Indicates that the object is virtual, that means it is a synthetic object introducedby the\nadapter for rendering purposes, e.g. an index range for large arrays.",
-						"Indicates that a data breakpoint is registered for the object."
+						"Deprecated: Indicates that a data breakpoint is registered for the object."
```
Comment (gregg-miskelly) [inline debugAdapterProtocol.json:3364]: I might suggest saying something like: "The 'hasDataBreakpoint' attribute should generally be used instead."

**Review by gregg-miskelly — APPROVED** [review]
Otherwise LGTM

**Review by gregg-miskelly — APPROVED** [review]
Otherwise LGTM

**Comment by weinand** [comment]
@isadorasophia thanks for the PR

**Pull request merged**